sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Properties/AssemblyInfo.cs

```diff
@@ -21,4 +21,4 @@
 [assembly: AssemblyTitle("Microsoft Azure SQL Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure SQL.")]
 [assembly: AssemblyVersion("3.0.0.0")]
-[assembly: AssemblyFileVersion("3.0.0.0")]
+[assembly: AssemblyFileVersion("3.1.0.0")]
```

Comment:
@ArthurMa1978 Since we need this SDK for PowerShell, which doesn't work yet with Track 2, and we need it to be ready by mid - July, we must generate Track 1 SDK for now.

Reply:
Ok, then please follow https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec to complete all steps